src/output-json-alert.c

```diff
@@ -217,6 +217,29 @@ void AlertJsonHeader(const Packet *p, const PacketAlert *pa, json_t *js)
     json_object_set_new(js, "alert", ajs);
 }
 
+static void AlertJsonTunnel(const Packet *p, json_t *js)
+{
+    json_t *tunnel = json_object();
+    if (tunnel == NULL)
+        return;
+
+    if (p->root == NULL) {
+        json_decref(tunnel);
+        return;
+    }
+
+    /* get a lock to access root packet fields */
+    SCMutex *m = &p->root->tunnel_mutex;
```

Comment:
It's not about the flow but the packet. I think the lock *might* be unnecessary, but I'm only interested in considering that if it actually shows up in profiling. Otherwise I prefer to keep things consistent, not in the least to assist code checkers like coverity.